pkg/doc/verifiable/credential_jwt.go

```diff
@@ -134,7 +134,7 @@ func (jcc *JWTCredClaims) refineCredFromJWTClaims() {
 	}
 
 	if jti := jcc.ID; jti != "" {
```

Comment:
I'm not sure I understand the purpose of these assignments? e.g., `jti := jcc.ID`

Reply:
Just give a shorter expressive name (they are standard JWT claim names).